proto/src/main/proto/Rss.proto

```diff
@@ -303,6 +303,7 @@ message ShuffleServerHeartBeatRequest {
   optional string gitCommitId = 23;
   optional int64 startTimeMs = 24;
   repeated ApplicationInfo applicationInfo = 25;
+  map<string, string> displayMetrics = 30;
```

Comment:
why not as the sequence number of 26?

Reply:
Sorry, there are some other fields in our internal patch, it is an accident.

Now, i fix it to 26, PTAL.